### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by luizcmarin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ By default, caching is disabled.
 
 ## Documentation
 
-- Guide: [English](docs/guide/en/README.md), [Russian](docs/guide/ru/README.md)
+- Guide: [English](docs/guide/en/README.md), [Português - Brasil](docs/guide/pt-BR/README.md), [Русский](docs/guide/ru/README.md)
```
Comment (vjik) [inline README.md:105]: broken PT-BR link

**Review by vjik — CHANGES_REQUESTED** [review]

File: docs/guide/pt-BR/image/algorithm.svg
Comment (roxblnfk) [inline docs/guide/pt-BR/image/algorithm.svg:1]: Shouldn't it be localized?

File: docs/guide/pt-BR/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    ]);
+$container = new Container($config);
+
+$getEngineName = static function (EngineInterface $engine) {
+    return $engine->getName();
+};
+
+$injector = new Injector($container);
+echo $injector->invoke($getEngineName);
+// outputs "Mark Two"
```
Comment (samdark) [inline docs/guide/pt-BR/README.md]: Should be translated.

Comment (luizcmarin) [reply docs/guide/pt-BR/README.md]: All codes are kept in the original, in English. Could you clarify the doubt?

File: docs/guide/pt-BR/image/algorithm.svg
Comment (luizcmarin) [reply docs/guide/pt-BR/image/algorithm.svg:1]: All imagens are kept in the original, in English. Could you clarify the doubt?

Comment (roxblnfk) [reply docs/guide/pt-BR/image/algorithm.svg:1]: For example schema in RU docs looks translated
image

File: docs/guide/pt-BR/README.md (hunk 2 above)
Comment (samdark) [reply docs/guide/pt-BR/README.md]: That's not code, that's comment in the code explaining it. Usually we're translating these.

Comment (luizcmarin) [reply docs/guide/pt-BR/README.md]: I understood. I will review all the packages

File: docs/guide/pt-BR/image/algorithm.svg
Comment (luizcmarin) [reply docs/guide/pt-BR/image/algorithm.svg:1]: I understood. I will review this

Comment (luizcmarin) [reply docs/guide/pt-BR/image/algorithm.svg:1]: done

File: docs/guide/pt-BR/README.md (hunk 2 above)
Comment (luizcmarin) [reply docs/guide/pt-BR/README.md]: done

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
:+1:

**Pull request merged**